keras_cv/models/resnet_v2_test.py

```diff
@@ -54,6 +54,13 @@ def test_application_variable_input_channels(self, app, last_dim, args):
     def test_model_can_be_used_as_backbone(self, app, last_dim, args):
         super()._test_model_can_be_used_as_backbone(app, last_dim, args)
 
+    @parameterized.parameters(*MODEL_LIST)
+    def test_model_serialization(self, app, last_dim, args):
+        super()._test_model_serialization(app, last_dim, args, "tf", "model")
```

Comment:
nit: maybe let's explicitly state the kwargs here since they're not immediately obvious (e.g. `save_format="tf"`